packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerItem.tsx

```diff
@@ -84,7 +84,7 @@ const StyledItem = styled('div', {
 `;
 
 const StyledItemLabel = styled.div`
-  font-weight: ${({ theme }) => theme.font.weight.medium};
+  font-weight: ${({ theme }) => theme.font.weight.mediumForNavigation};
```

Comment:
You can remove this change (the issue is that font.weight.medium definition is wrong)